metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_audit.py

```diff
@@ -84,7 +84,19 @@ class BigqueryTableIdentifier:
 
```

Comment:
@treff7es do you know what the `$` is doing in `_BIGQUERY_DEFAULT_SHARDED_TABLE_REGEX`? Seems like we'd support `[PREFIX]$YYYYMMDD` but `$` is in the `invalid_chars` set

Reply:
From what I understand that $ in the middle is for matching non sharded tables. but this is really complex can be simplified.